Guides/Development.md

```diff
@@ -30,6 +30,8 @@ Run `swift build` or simply `make` in the project's root directory.
 
 If you add symbols you may need to run `make exports` which will generate [Sources/MongoSwiftSync/Exports.swift](Sources/MongoSwiftSync/Exports.swift). This makes symbols declared in `MongoSwift` available to importers of `MongoSwiftSync`.
 
+Note: Do not run `make clean`.
+
 ### In Xcode
 We do not provide or maintain an already-generated `.xcodeproj` in our repository. Instead, you must generate it locally.
```

Comment:
When I tried to follow these Xcode directions, I got some messages that this was not necessary anymore, I also never really got Xcode working fully. (I would get errors about weird stuff when trying to build)